<?xml version="1.0"?>
<launch>
  <arg name="model_name" value="uav"/> 
  <node name="virtual_environment" pkg="navigation" type="ModelState" output="screen">
    <param name="model_name" type="string" value="$(arg model_name)" />
   <!-- <remap from="/pose" to="/odometry"/> -->
	<remap from="/pose" to="/uav/pose"/>
    <remap from="/set_model_state" to="/gazebo/set_model_state"/>
  </node>
  <param name="use_sim_time" value="false" />
</launch>
